tests/AutoReview/ProjectCodeTest.php

```diff
@@ -43,7 +43,9 @@ final class ProjectCodeTest extends TestCase
      */
     private static $classesWithoutTests = [
         \PhpCsFixer\Console\SelfUpdate\GithubClient::class,
+        \PhpCsFixer\Console\Command\DocumentationCommand::class,
         \PhpCsFixer\Doctrine\Annotation\Tokens::class,
+        \PhpCsFixer\Documentation\DocumentationGenerator::class,
```

Comment:
I added these classes here because I'm not sure testing them really has value. I mean, if they fail somehow, we should see it with docs being out of date or having unexpected changes. Shall I add tests anyway?

Reply:
No answers, I assume this is considered OK.